src/main.js

```diff
@@ -5,16 +5,25 @@
 function convertItemsToUnorderedList(listItems) {
   const ulElement = document.createElement("ul");
   listItems.forEach(item => {
+    const handler = document.createElement("span");
+    handler.classList.add("handler");
+
     const liElement = document.createElement("li");
     liElement.classList.add("fs-api-entry", `fs-api-${item.type}`);
+    if (item.type === "directory") {
+      handler.textContent = "›";
+      handler.classList.add("handler");
```

Comment:
The classname `handler` is too generic. We should prefer something like `fs-api-directory-handler`.